Client.xcodeproj/project.pbxproj

```diff
@@ -10673,7 +10673,6 @@
 				ENABLE_BITCODE = NO;
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(inherited)",
-					"$(BUILD_DIR)/Release$(EFFECTIVE_PLATFORM_NAME)",
```

Comment:
Womndering why do we need the project settings changes ?

Reply:
These build directories don't exist in Debug builds, so they give warnings

Reply:
These removals are only to Debug configuration specifically :)